linkup-cli/src/background_tunnel.rs

```diff
@@ -6,10 +6,11 @@ use std::thread;
 use std::time::Duration;
 
 use daemonize::{Daemonize, Outcome};
+use nix::sys::signal::Signal;
```

Comment:
Wrong signal? Should it be nix?

Reply:
Well, this is the currently one in master, so I thought so 😅 
https://github.com/mentimeter/linkup/blob/main/linkup-cli/src/signal.rs

But I'm not quite sure 🤔